app/models/response_plan.rb

```diff
@@ -39,6 +39,14 @@ class ResponsePlan < ActiveRecord::Base
   validates :person, presence: true
   validate :errors_from_associated_person
 
+  def self.drafts
```

Comment:
Since we have a lot of logic dependent on the state of a response plan, it may make sense to break it out into multiple models using [STI].

[STI]: http://api.rubyonrails.org/classes/ActiveRecord/Base.html#class-ActiveRecord::Base-label-Single+table+inheritance